pkgs/by-name/re/renode-bin/package.nix

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
I believe renaming the recipe should be the first commit to minimize changes. Also, please include the unstable binary recipe so we keep both.